app/server/src/controllers/projectController.ts

```diff
@@ -140,5 +140,21 @@ export default (config) => {
         sendSuccess(response, null);
       });
     },
+
+    async addSamples(request, response, next) {
```

Comment:
Can we also get rid of the old singular endpoint?

Reply:
i just left just incase we need it but will remove makes sense